### PR TITLE
Ft/add new branch

### PR DESCRIPTION
**Pull request by Samantha-KY**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+}
+.body__card__paragraph{
+    text-align: center;
+}
+.body__card__footer{
+    text-align: center;
+}
+.body__card__item__img{
+
+}
```
Comment (hirwaaldo1) [inline style.css]: remove this class in css because it does not have style

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]

**Review by hirwaaldo1 — APPROVED** [review]
it looks good

File: flexing.html [hunk 2]
```diff
@@ -4,35 +4,154 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
     <title>Document</title>
```
Comment (edson-pro) [inline flexing.html]: change the title no meaning title

File: flexing.html [hunk 3]
```diff
@@ -4,35 +4,154 @@
...
-                <li>Home</li>
-                <li>About us</li>
-                <li>Service information goes here</li>
-                <li>Blog</li>
-                <li>Contact</li>
-            </ul>
-        </nav>
-    </header>
+<body >
+    <div class="navigate">
```
Comment (edson-pro) [inline flexing.html]: remove navigation class

File: flexing.html [hunk 4]
```diff
@@ -4,35 +4,154 @@
...
+            <path d="M5 0.5H157C159.485 0.5 161.5 2.51472 161.5 5V47C161.5 49.4853 159.485 51.5 157 51.5H5C2.51472 51.5 0.5 49.4853 0.5 47V5C0.5 2.51472 2.51472 0.5 5 0.5Z" stroke="#1BA0AB"/>
+            </svg>     
+        </div>
+</div>
+        <div class="body">
```
Comment (edson-pro) [inline flexing.html]: remove body classs

File: flexing.html [hunk 5]
```diff
@@ -4,35 +4,154 @@
...
+                            </svg>  
+                    </div> 
+                  </div>
+            </div>
+        </div>
+    </div>
+    <div class= "body__footer">
```
Comment (edson-pro) [inline flexing.html]: remove body prefix

File: flexing.html [hunk 6]
```diff
@@ -4,35 +4,154 @@
...
-                <li>Contact</li>
-            </ul>
-        </nav>
-    </header>
+<body >
+    <div class="navigate">
+    <nav class="nav">
+        <h1 class="nav__text1" >Medical Core</h1>
+        <div class="nav__lists">
+            <p  class="nav__list1 nav__lists--place">Home</p>
```
Comment (edson-pro) [inline flexing.html]: nav__lists--place modifier has no reason to use it

File: flexing.html [hunk 7]
```diff
@@ -4,35 +4,154 @@
...
+                            <path fill-rule="evenodd" clip-rule="evenodd" d="M0.680771 0.680771C0.737928 0.623469 0.805828 0.578007 0.880583 0.546987C0.955337 0.515967 1.03548 0.5 1.11641 0.5C1.19735 0.5 1.27749 0.515967 1.35224 0.546987C1.427 0.578007 1.4949 0.623469 1.55205 0.680771L8.9358 8.06452C8.9931 8.12168 9.03857 8.18958 9.06959 8.26433C9.10061 8.33909 9.11657 8.41923 9.11657 8.50016C9.11657 8.5811 9.10061 8.66123 9.06959 8.73599C9.03857 8.81074 8.9931 8.87864 8.9358 8.9358L1.55205 16.3196C1.43651 16.4351 1.27981 16.5 1.11641 16.5C0.953015 16.5 0.79631 16.4351 0.680771 16.3196C0.565232 16.204 0.500322 16.0473 0.500322 15.8839C0.500322 15.7205 0.565232 15.5638 0.680771 15.4483L7.63011 8.50016L0.680771 1.55205C0.623469 1.4949 0.578006 1.427 0.546986 1.35224C0.515967 1.27749 0.5 1.19735 0.5 1.11641C0.5 1.03548 0.515967 0.955338 0.546986 0.880583C0.578006 0.805829 0.623469 0.737928 0.680771 0.680771Z" fill="#1BA0AB"/>
+                            </svg>
+                            </h3>
+                </div>      
+        </div>
+        <div class="body__team">
```
Comment (edson-pro) [inline flexing.html]: remove body prefix

**Review by edson-pro — CHANGES_REQUESTED** [review]
format your code and make it quick

**Review by edson-pro — APPROVED** [review]
extravagant

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+.navigate__wrapper {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: antiquewhite;
+}
+
+.navv {
```
Comment (karemarsy) [inline src/style.css]: why did you name this class `navv`? your naming convention should be clear to everyone

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,316 @@
...
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2px;
+    color: #737373;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+
```
Comment (karemarsy) [inline src/style.css]: remove spaces

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,316 @@
...
+    background: #FFF5F0;
+    flex: none;
+    order: 6;
+    flex-grow: 0;
+    padding-top: 5rem;
+    margin-top: 5rem;
+
+}
+
+.footer-section {
```
Comment (karemarsy) [inline src/style.css]: why didn't use this `footer-section` as an element of `footer`

**Review by karemarsy — COMMENTED** [review]
do some cleanup

**Review by karemarsy — APPROVED** [review]

**Pull request merged**